src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -1431,6 +1456,10 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
       context: transactionEventContext
     });
     await vm.runTx({
+      skipHardForkValidation: true,
```

Comment:
We don't need to check these since we've already run all these transactions before, these validations shouldn't be needed.